libs/eo/help/shell/src/lib/eo-geography-page.component.ts

```diff
@@ -15,12 +15,192 @@
  * limitations under the License.
  */
 import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { EoStackComponent } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { eoRoutes } from '@energinet-datahub/eo/shared/utilities';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
+  imports: [EoStackComponent, RouterModule],
   selector: 'eo-geography-page',
-  styles: [``],
-  template: ` <div>Geography</div> `,
+  styles: [
+    `
+      :host {
+        display: grid;
+        grid-auto-flow: column;
```

Comment:
Mixing auto-flow column with a defined column doesn't make sense.
This can likely just be removed.